docs/connectors/postgresql.asciidoc

```diff
@@ -1005,6 +1005,41 @@ The last option for `decimal.handling.mode` configuration property is `string`.
 
 PostgreSQL supports `NaN` (not a number) special value to be stored in the `DECIMAL`/`NUMERIC` values. Only `string` and `double` modes are able to handle such values encoding them as either `Double.NaN` or string constant `NAN`.
 
+[[hstore-values]]
+==== HStore values
+
+When `hstore.handling.mode` configuration property is set to `map`, then the connector will use the `java.util.Map<String,String>` logical type for all `HSTORE` columns. This is the default mode.
+
+[cols="15%a,15%a,35%a,35%a",width=100,options="header,footer",role="table table-bordered table-striped"]
+|=======================
+|PostgreSQL Data Type
+|Literal type (schema type)
+|Semantic type (schema name)
+|Notes
+
+|`HSTORE`
+|`hstore`
+|`java.util.Map<String,String>`
+|
+
+|=======================
+
+However, when `hstore.handling.mode` configuration property is set to `json`, then the connector will represent all `HSTORE` values as JSON String values and encodes them as follows:
+
+[cols="15%a,15%a,35%a,35%a",width=100,options="header,footer",role="table table-bordered table-striped"]
+|=======================
+|PostgreSQL Data Type
+|Literal type (schema type)
+|Semantic type (schema name)
+|Notes
+
+|`HSTORE`
+|`hstore`
```

Comment:
The `schema type` is `STRING` in this case. And the semantic type should be `io.debezium.data.Json`. This is actually still missing from the implementation, I'll update that in the code right now.